common/app/views/fragments/inlineSvg.scala.html

```diff
@@ -1,4 +1,4 @@
-@(file: String, path: String = "", classes: Seq[String] = List())
-<span class="inline-@file @if(path){inline-@path} @classes.mkString(" ")">
+@(file: String, path: String = "", classes: Seq[String] = List(), attributes: Map[String, String] = Map())
```

Comment:
What attributes other than `title` are we expecting? Maybe we could just use `title: Option[String] = None`.

Map params get messy real quick.